modules/nf-core/peddy/main.nf

```diff
@@ -0,0 +1,53 @@
+process PEDDY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::peddy=0.4.8"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/peddy:0.4.8--pyh5e36f6f_0' :
+        'quay.io/biocontainers/peddy:0.4.8--pyh5e36f6f_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(vcf_tbi)
+    path ped
+
+    output:
+    tuple val(meta), path("*.html")     , emit: html
+    tuple val(meta), path("*.csv")      , emit: csv
+    tuple val(meta), path("*.peddy.ped"), emit: ped
+    tuple val(meta), path("*.png")      , optional: true, emit: png
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
`prefix` doesn't seem to be used in the module. 
I would add the line `--prefix $prefix` to the peddy command.